CHANGELOG.md

```diff
@@ -3,3 +3,6 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+BATIAI-2164-cross-alb:
```

Comment:
Do not directly modify the Changelog file

Reply:
comment in change log file removed